src/charm.py

```diff
@@ -90,6 +88,8 @@ def __init__(self, *args):
             external_url=self._external_url,
             refresh_event=[self.certificates.on.certificate_available],
         )
+
+        # The self_profiling_endpoint_provider enables a remote Parca to scrape profiles from this Parca instance.
```

Comment:
I think this should be in charmcraft.yaml in the endpoint description, not as a code comment